### ISSUE TITLE
Startup class not getting invoked in Durable function

### ISSUE THREAD

**Issue by VenkateshSrini** [BorisWilhelms/azure-function-dependency-injection#20]
Hi,
I have used your extension for DI. But I'm using it in Durable function. Will the startup classnot be invoked in durable function?

**Comment by rasmuschristensen** [comment]
maybe you ran into the issue running .netcore 2.1 and not 2.0. See the previous issue

**Comment by VenkateshSrini** [comment]
Yeah I guess that was an issue with Azure function runtime team. They broke something while doing the re-targeting the runtime to .NEt core 2.0

**Comment by BorisWilhelms** [comment]
@VenkateshSrini is this issue still relevant? I have never tried the DI solution in Durable functions.

**Comment by VenkateshSrini** [comment]
Yes this is very much relevant. We might need to persist additional information as part of workflow creation which might be needed in successive steps. I have tried it using WilliZone DI extension, but definitely would need this to be part of Azure function runtime itself. Whether it is normal function or Durable function these injected components are part of functionality
